src/darker/verification.py

```diff
@@ -67,3 +67,39 @@ def verify_ast_unchanged(
     except AssertionError as exc_info:
         debug_dump(black_chunks, edited_linenums)
         raise NotEquivalentError(str(exc_info))
+
+
+class ASTVerifier:
+    """
+    Verifies if reformatted TextDocument is AST-equivalent to baseline.
+
+    Keeps in-memory data about previous comparisons to improve performance.
+    """
```

Comment:
I'd like to use the same docstring convention everywhere: no newline before first line, end with a blank line.

```suggestion
    """Verify that reformatted TextDocument is AST-equivalent to baseline

    Keeps in-memory data about previous comparisons to improve performance.

    """
```

Reply:
Sure, this is just trivial.